packages/ui/src/pages/DashboardPage/components/AddSession.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { defineEmits } from "vue";
+
+const emit = defineEmits(["add-session"]);
+
+const addSession = () => {
+  emit("add-session");
+};
+</script>
+
+<template>
+  <div
```

Comment:
I would make this whole thing clickable instead of just the icon and text. Just make sure the cursor turns into a pointer on the whole thing, and you don't need the text to be in a `<button>`